features/step_definitions/networking.rb

```diff
@@ -1008,3 +1008,13 @@
   end 
   logger.info "The multicast is enable in the #{project_name} project"
 end
+
+Given /^I install load-sctp-module/ do
+  ensure_admin_tagged
+  _admin = admin
+  @result = _admin.cli_exec(:get, resource: "machineconfigs", output: "jsonpath={.items[10].metadata.name}")
```

Comment:
`items[10]` won't work forever.  We should be able to match the array element based on the `metadata.name`

We have to use single quotes, or escape the double quotes
```go
'jsonpath={.items[?(@.metadata.name=="load-sctp-module")].metadata.name}'
```
should work

Reply:
Static indices are discouraged, yep

Reply:
@rbbratta Fixed it, thanks for good tip!